### ISSUE TITLE
My iphone 6s has black screen with checkra1n and iphone logo.

### ISSUE THREAD

**Issue by Speedruner08** [foxlet/bootra1n#138]
So, when i did jailbreak using this tool called "Bootra1n", mine iphone 6s did show up checkra1n loding screen, but then the screen went completly black, now when i turn on mine iphone 6s that checkra1n screen shows up, but then i get that black screen again. I tried restoring ispw, but i can not because mine pc does not detect it in itunes. Please can someone help me or is mine iphone 6s bricked forever?? Edit: I was using ios 14.6.

**Comment by StupidRepo** [comment]
I don't think it works on iPhone 6 and below. It only works on iPhone 7 and above I think. Sorry bud.
